numpy/core/src/common/npy_cpu_features.c.src

```diff
@@ -539,6 +553,12 @@ npy__cpu_init_features_linux(void)
             close(fd);
         }
     }
+    #else
+    hwcap = getauxval(NPY__HWCAP);
+    #ifdef __arm__
+    hwcap2 = getauxval(NPY__HWCAP2);
+    #endif
+    #endif
     if (hwcap == 0 && hwcap2 == 0) {
```

Comment:
`get_feature_from_proc_cpuinfo()` should be limited only for linux since freebsd doesn't have `/proc/cpuinfo`